qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -1484,15 +1488,14 @@ def set_html(self, html, base_url=QUrl()):
         # percent encoded content is 2 megabytes minus 30 bytes.
         self._widget.setHtml(html, base_url)
 
-    def _show_error_page(self, url, error):
+    def _show_error_page(self, url: QUrl, error: str) -> None:
         """Show an error page in the tab."""
         log.misc.debug("Showing error page for {}".format(error))
-        url_string = url.toDisplayString()
-        error_page = jinja.render(
-            'error.html',
-            title="Error loading page: {}".format(url_string),
-            url=url_string, error=error)
-        self.set_html(error_page)
+        error_url = QUrl("qute://error")
+        queries = QUrlQuery()
+        queries.setQueryItems((("url", url.toDisplayString()), ("error", error)))
+        error_url.setQuery(queries)
+        self.load_url(error_url)
```

Comment:
The additional load (rather than `set_html`) seems to break the red URL color in the statusbar on errors - so I guess `AbstractTab._update_load_status` should probably always return `usertypes.LoadStatus.error` if we just loaded an error page?